packages/dd-trace/src/appsec/blocking.js

```diff
@@ -111,6 +111,10 @@ function block (req, res, rootSpan, abortController, actionParameters) {
 
   const { body, headers, statusCode } = getBlockingData(req, null, rootSpan, actionParameters)
 
+  for (const headerName of res.getHeaderNames()) {
+    res.removeHeader(headerName)
```

Comment:
Could there be problems if we remove proxy headers, cors or similar? although we are blocking an attack...

Reply:
yes, but we talked about this with the rest of the libs, and that's what has been decided, there is even a system test about it. If you think that's wrong, go ahead and open a new discussion with the libs